tests/flow/test_generic_cmds.py

```diff
@@ -90,169 +87,173 @@ def test_evict():
     with env.getClusterConnectionIfNeeded() as r:
         init(env, r)
         
-        assert r.execute_command('RENAME', 't1', 't4')
-        res = r.execute_command('ts.info', 't3')
+        assert r.execute_command('RENAME', 't{1}', 't{1}_renamed')
+        res = r.execute_command('ts.info', 't{1}_agg')
         index = res.index(b'sourceKey')
-        env.assertEqual(res[index+1], b't4')
+        env.assertEqual(res[index+1], b't{1}_renamed')
 
         res = r.execute_command('TS.QUERYINDEX', 'name=(mush,zavi,rex)')
-        env.assertEqual(sorted(res), [b't2', b't3', b't4'])
-        res = r.execute_command('keys *')
-        env.assertEqual(sorted(res), [b't2', b't3', b't4'])
-
-        assert r.execute_command('RENAME', 't3', 't1')
-        res = r.execute_command('ts.info', 't4')
+        env.assertEqual(sorted(res), sorted([b't{2}', b't{1}_agg', b't{1}_renamed']))
+        res = r.execute_command('TS.MGET', 'filter',  'name=(mush,zavi,rex)')
+        env.assertEqual(sorted(res), sorted([[b't{2}', [], []],
+                                             [b't{1}_agg', [], []],
+                                             [b't{1}_renamed', [],  [10, b'19']]]))
+
+        assert r.execute_command('RENAME', 't{1}_agg', 't{1}')
+        res = r.execute_command('ts.info', 't{1}_renamed')
         index = res.index(b'rules')
-        env.assertEqual(res[index+1], [[b't1', 10, b'AVG']])
+        env.assertEqual(res[index+1], [[b't{1}', 10, b'AVG']])
 
 
 def test_renamenx():
     env = Env()
-    env.skipOnCluster()
     with env.getClusterConnectionIfNeeded() as r:
         init(env, r)
         
-        assert r.execute_command('RENAMENX', 't1', 't4')
-        res = r.execute_command('ts.info', 't3')
+        assert r.execute_command('RENAMENX', 't{1}', 't{1}_renamed')
+        res = r.execute_command('ts.info', 't{1}_agg')
         index = res.index(b'sourceKey')
-        env.assertEqual(res[index+1], b't4')
+        env.assertEqual(res[index+1], b't{1}_renamed')
 
         res = r.execute_command('TS.QUERYINDEX', 'name=(mush,zavi,rex)')
-        env.assertEqual(sorted(res), [b't2', b't3', b't4'])
-        res = r.execute_command('keys *')
-        env.assertEqual(sorted(res), [b't2', b't3', b't4'])
+        env.assertEqual(sorted(res), [b't{1}_agg', b't{1}_renamed', b't{2}'])
+        res = r.execute_command('TS.MGET', 'filter', 'name=(mush,zavi,rex)')
+        env.assertEqual(sorted(res), [[b't{1}_agg', [], []], [b't{1}_renamed', [], [10, b'19']], [b't{2}', [], []]])
 
-        assert r.execute_command('RENAMENX', 't3', 't1')
-        res = r.execute_command('ts.info', 't4')
+        assert r.execute_command('RENAMENX', 't{1}_agg', 't{1}_agg_renamed')
+        res = r.execute_command('ts.info', 't{1}_renamed')
         index = res.index(b'rules')
-        env.assertEqual(res[index+1], [[b't1', 10, b'AVG']])
+        env.assertEqual(res[index+1], [[b't{1}_agg_renamed', 10, b'AVG']])
 
 
 def test_copy_compressed():
```

Comment:
@danni-m  did you forgot to delete this func? Cause it now looks like the func above.